controllers/Package.js

```diff
@@ -26,14 +26,15 @@ exports.getOnePackage = (req, res) => {
 }
 
 exports.getAllPackages = (req, res) => {
-  Package.find(
-    {},
-    {
-      'iternary._id': 0,
-    },
-    function(err, allpackages) {
+  Package.find({})
+    .select({
+      _id: 1,
+      name: 1,
+      description: 1,
+      images: 1,
+    })
+    .exec(function(err, allpackages) {
```

Comment:
You're using `then` in one place and `exec` in the next. Let's try to stick to then itself, we can easily shift to async/await in the future.